### ISSUE TITLE
Index only adds 12 items at a time (Python)

### ISSUE THREAD

**Issue by leobaumgardt** [unum-cloud/usearch#7]
Hi Ash,

great library!

I'm having some trouble:

My indices contain fewer items than I add.

In fact, I'm having this issue even when running the sample Python code from your README.md.

```
import numpy as np
import usearch

index = usearch.Index(
    dim=256, # Define the number of dimensions in input vectors
    metric='cos', # Choose the "metric" or "distance", default = 'ip', optional
    dtype='f16', # Quantize to 'f16' or 'i8q100' if needed, default = 'f32', optional
    connectivity=16, # How frequent should the connections in the graph be, optional
    expansion_add=128, # Control the recall of indexing, optional
    expansion_search=64, # Control the quality of search, optional
)

n = 100
labels = np.array(range(n), dtype=np.longlong)
vectors = np.random.uniform(0, 0.3, (n, index.ndim)).astype(np.float32)

# check for duplicate vectors
vectors_deduped = np.unique(vectors, axis=0)
assert len(vectors_deduped) == len(vectors), "Duplicate vectors found"

# ensure we have n labels and vectors
assert len(vectors) == len(labels) == n, "Number of vectors and labels must match"

# You can avoid copying the data
# Handy when build 1B+ indexes of memory-mapped files
index.add(labels, vectors, copy=True)
assert len(index) == n, f"Index should contain {n} items, but contains {len(index)}"
```

Output:

`AssertionError: Index should contain 100 items, but contains 12``

Curiously:

- I'm getting 12 items, no matter whether I add 100 items or 1_000. If I add 12 or fewer, the assertion passes. As soon as I pass 13, only 12 are added.
- If I re-run the `index.add()` line, the index size increments by 12 every time.
- I tried initializing the index with `capacity=100`, but that also didn't change things.
- The parameter `copy=True` also apparently doesn't influence this behavior. I tried it both True and False.

My questions:

1. What am I doing wrong? 

2. What does copy=True actually do? I didn't find any further documentation. It allows the index to just reference the label/vector data as long as it's stored in, say, an numpy array? Also, does copy=True mean "Yes, copy the data", or does it mean "Avoid copying the data"? Since you're explicity passing it when stating "You can avoid copying the data", it's a bit ambiguous.

Thank you!

**Comment by ashvardanian** [comment]
Yes, you are right, there is a bug in the Python wrapper layer. I noticed it yesterday, will share a patch later today. 

Copy is True by default. If you pass False, the index will be storing a pointer to some external vector. It’s useful when you want to construct an index of a memory-mapped binary file and you don’t have enough RAM for both index and vectors.

**Comment by ashvardanian** [comment]
It was solved with [caee311](https://github.com/unum-cloud/usearch/commit/caee311422294d8193e8e608b78624c343a82d3c). The new image should reach PyPi in 30m.
